x-pack/elastic-agent/pkg/agent/cmd/enroll.go

```diff
@@ -28,8 +28,8 @@ import (
 func newEnrollCommandWithArgs(_ []string, streams *cli.IOStreams) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "enroll",
-		Short: "Enroll the Agent into Fleet",
-		Long:  "This will enroll the Agent into Fleet.",
+		Short: "Enroll the Agent into Fleet Server",
```

Comment:
I don't really like the name of this. You are enrolling the Elastic Agent into all of Fleet. The Kibana piece matters as well its all connected. Just because it talks to Fleet Server is only a piece of the puzzle.

To me it's all "Fleet".

Reply:
Yeah, is is exactly the part I'm torn. I wonder how the perspective here will be for a new user that just accessed Fleet in the UI. Is it clear that enrollment happens into another component?

Reply:
I would also prefer 'Fleet'. Users interact with Fleet as a single environment/product. Fleet Server is just a detail and most users should not even be concerned with its existence.